.makim.yaml

```diff
@@ -73,16 +73,17 @@ groups:
       smoke:
         help: Run smoke tests
         dependencies:
-          - task: smoke-tests.simple
-          - task: smoke-tests.complex
-          - task: smoke-tests.containers
-          - task: smoke-tests.unittest
-          - task: smoke-tests.vars-env
-          - task: smoke-tests.test-vars
-          - task: smoke-tests.bash
-          - task: smoke-tests.dir-absolute-path
-          - task: smoke-tests.dir-no-path
-          - task: smoke-tests.dir-relative-path
+          # - task: smoke-tests.simple
```

Comment:
Any reason for disabling all thse tests?

Reply:
No...I just testing the interactive-args...so I comment down other test cases and forget to un-comment them before committing the changes. Sorry..

Reply:
I have commit the new changes.